frontend/package.json

```diff
@@ -86,7 +87,7 @@
     "spacetime": "^7.4.0",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
-    "typescript": "4.4.3",
+    "typescript": "^4.5.5",
```

Comment:
Could you please separate this into it's own commit as well? For debugging ease later.

I assume the upgrade is necessary for material-react-table to work?

Reply:
yes its needed for the material-react-table, and I don't think it should be a separate commit, it won't compile on its own

Reply:
I think there was a misunderstanding. I tried upgrading typescript to 4.5.5 on main branch by itself and it does build, and run. The tests and tsc pass too.

Reply:
typescript bump is now a separate commit

Reply:
Thanks